os/kernel/init/os_bringup.c

```diff
@@ -290,6 +294,15 @@ static inline void os_do_appstart(void)
 	prodconfig_register();
 #endif
 
+#ifdef CONFIG_LOG_DUMP
+	svdbg("Starting log_dump thread\n");
+
+	pid = kernel_thread(LOG_DUMP_NAME, 200, LOG_DUMP_STACKSIZE, log_dump, NULL);
```

Comment:
How about 100 or lower for priority? It's not most urgent work so that 200 looks too much high I think.

Reply:
1. We are using the thread after each 4k log array is filled to compress.